api.rst

```diff
@@ -363,6 +363,10 @@ The client can call it by posting an object like
 
 The keys of the object match the parameter names. Note that PostgreSQL converts parameter names to lowercase unless you quote them like :sql:`CREATE FUNCTION foo("mixedCase" text) ...`.
 
+PostgreSQL has four procedural languages that are part of the core distribution: PL/pgSQL, PL/Tcl, PL/Perl, and PL/Python. There are many other procedural languages distributed as additional extensions. Also, plain SQL can be used to write functions (as shown in the example above).
+
+By default, a function is to be executed with the privileges of the user that calls it. This means, that the user has to have all permissions to do all operations the procedure performs. But if the function was defined with `SECURITY DEFINER` options, only one permission check will take place – the permission to call the function. See [PostgreSQL documentation](https://www.postgresql.org/docs/current/static/sql-createfunction.html) for more details.
```

Comment:
Because the new docs are in rst format rather than markdown, you'll need to adjust the code literals and links a little. For instance

```rst
:code:`SECURITY DEFINER`
```
and
```rst
`PostgreSQL documentation <https://www.postgresql.org/docs/current/static/sql-createfunction.html>`_
```

The syntax is kind of annoying but it supports more reliably linking between internal pages.